### ISSUE TITLE
ML: support .modc export to frozen graph

### ISSUE THREAD

**Issue by paulmueller** [DC-analysis/dclab#84]
Forked from #78.

1. For OpenCV (frozen graph): https://github.com/ZELLMECHANIK-DRESDEN/dclab/issues/78#issuecomment-712454867 Does it work with C++/OpenCV?
2. What is this neuropod? https://github.com/uber/neuropod
3. Have a look at keras2cpp https://github.com/pplonski/keras2cpp (used in https://dx.doi.org/10.1038/s41592-020-0831-y)

**Comment by maikherbig** [comment]
I prepared some functions which allow you to:
- Pre-process raw rtdc images (cropping, padding, zooming, normalization)
- Forward these images through a frozen neural net using OpenCV's dnn module (NN's can be converted to a frozen graph using AID)

Currently, I'm working on a tutorial:
https://github.com/maikherbig/AIDeveloper/tree/master/Tutorial%20Deploy%20to%20OpenCV%20dnn
It already covers the main aspect, which is how to pre-process images and forward them through a frozen model. The main script 
[aid_cv2_dnn.py](https://github.com/maikherbig/AIDeveloper/blob/master/Tutorial%20Deploy%20to%20OpenCV%20dnn/aid_cv2_dnn.py) which does the heavy lifting is done and ready. In the next days I will add test functions and more explanation how you can export models using AID to the tutorial.

**Comment by paulmueller** [comment]
See #236
